src/components/table/table-row/table-row-modern.style.js

```diff
@@ -1,30 +1,79 @@
+import { css } from 'styled-components';
```

Comment:
Just noticed that we have `table-row-modern`
What I see we have two types of files:
- `component-classic.style.js` (for legacy styles)
- `component.style.js` (for dls styles)
Can you change the `table-row-modern.style.js` to `table-row.style.js` to keep convention please 😃

Reply:
That is indeed the naming convention we generally use.

But in this case, we actually have 3 `style` files:

* [`table-row.style.js`](https://github.com/Sage/carbon/blob/42fda8d/src/components/table/table-row/table-row.style.js)  –  This imports from `table-row-modern.style.js` and `table-row-classic.style.js`
* [`table-row-modern.style.js`](https://github.com/Sage/carbon/blob/42fda8d/src/components/table/table-row/table-row-modern.style.js)
* [`table-row-classic.style.js`](https://github.com/Sage/carbon/blob/42fda8d/src/components/table/table-row/table-row-classic.style.js)

So I can't rename the `table-row-modern.style.js` file to `table-row.style.js`, because we already have a file called `table-row.style.js`.